build/yaml/crd/nsx.vmware.com_subnets.yaml

```diff
@@ -0,0 +1,106 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: subnets.nsx.vmware.com
+spec:
+  group: nsx.vmware.com
+  names:
+    kind: Subnet
+    listKind: SubnetList
+    plural: subnets
+    singular: subnet
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: Subnet is the Schema for the subnets API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: SubnetSpec defines the desired state of Subnet
+            properties:
+              accessMode:
+                default: private
+                description: Access mode of subnet, accessible only from within VPC
+                  or from outside VPC. Defaults to private.
+                enum:
+                - private
+                - public
+                - isolated
+                type: string
+              ipAddresses:
+                description: Subnet CIDR.
+                items:
+                  type: string
+                maxItems: 2
+                minItems: 0
+                type: array
+              ipv4SubnetSize:
+                default: 64
```

Comment:
The default subnet prefix is /28, which is different with size 64 here @lxiaopei

Reply:
> The default subnet prefix is /28, which is different with size 64 here @lxiaopei

If the default prefix is /28, then default `ipv4SubnetSize` should be 16? @lxiaopei , could you help confirm?

Reply:
It seems that the default value is still 64 on api doc of nsx-t 4.1.0, not sure if I'm using the right version.

Reply:
For VPC Subnet API, the default subnet size is 64.  
I set the default subnet prefix in VPC NetworkConfiguration CR as 28 according to current T1 NSXNetworkConfig CR.
Do we need to keep them the same value?

Reply:
Also changed default subnet prefix in VPC NetworkConfiguration CR  to 26 to be consistent with the default subnet size 64 here.